cli/src/semgrep/formatter/sarif.py

```diff
@@ -414,6 +414,7 @@ def format(
         - Full specification is at:
             https://docs.oasis-open.org/sarif/sarif/v2.1.0/cs01/sarif-v2.1.0-cs01.html
         """
+        sorted_findings = sorted(rule_matches)
```

Comment:
this seems fine, but curious why we added it (and maybe leave a comment explaining why)

Reply:
In osemgrep we use the sorted json output to build the sarif output. IIRC there is a comment somewhere that this is done so the output is deterministic in the face of parallelism. I can add a comment in the code on Monday (anyone feel free to do so before then). It should be possible to rewrite it to not use the json output in osemgrep and thus not sort.